src/otx/core/model/anomaly.py

```diff
@@ -402,22 +330,17 @@ def export(
         Returns:
             Path: path to the exported model.
         """
-        min_val = self.normalization_metrics.state_dict()["min"].cpu().numpy().tolist()
-        max_val = self.normalization_metrics.state_dict()["max"].cpu().numpy().tolist()
-        image_shape, mean_values, scale_values = self._get_values_from_transforms()
-        exporter = _AnomalyModelExporter(
-            image_shape=image_shape,
-            image_threshold=self.image_threshold.value.cpu().numpy().tolist(),
-            pixel_threshold=self.pixel_threshold.value.cpu().numpy().tolist(),
-            task=self.task,
-            mean_values=mean_values,
-            scale_values=scale_values,
-            normalization_scale=max_val - min_val,
-        )
-        return exporter.export(
+        if export_format == OTXExportFormatType.OPENVINO:
```

Comment:
When I tried to make exportable_code, it works on anomaly classification. Are there reasons not to support it?

Reply:
The answer is simple. We don't have demo with anomaly. It produces zip archive of course, but we can't run anomaly there